src/storage/btree.c

```diff
@@ -9186,6 +9176,26 @@ btree_index_capacity (THREAD_ENTRY * thread_p, BTID * btid, BTREE_CAPACITY * cpc
   ret = btree_get_subtree_capacity (thread_p, &btid_int, root, cpc, &stats_env);
   btree_scan_clear_key (&stats_env.btree_scan);
 
+  if (cpc->dis_key_cnt > 0)
+    {
+      cpc->avg_val_per_dedup_key = (int) (cpc->tot_val_cnt / cpc->deduplicate_dis_key_cnt);
```

Comment:
cpc->dis_key_cnt > 0 조건으로 cpc->deduplicate_dis_key_cnt == 0 인 경우는 없어 보이나 안전 장치로 assert()를 추가해 두는게 어떨까요?

```suggestion
      assert (cpc->deduplicate_dis_key_cnt != 0);
      cpc->avg_val_per_dedup_key = (int) (cpc->tot_val_cnt / cpc->deduplicate_dis_key_cnt);
```

Reply:
`assert(cpc->deduplicate_dis_key_cnt > 0);`를 추가하겠습니다.